### PR TITLE
Updated npm copyhooks command to make git hooks os independent

### PR DESCRIPTION
**Pull request by neerajshukla1911**
Updated npm copyhooks command to make git hooks os independent.

Jira Ticket: https://browserstack.atlassian.net/browse/ACE-1378

### CODE REVIEW

File: scripts/copyhooks.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+const srcDir = './hooks/';
+const dstDir = './.git/hooks/'
+
+fs.readdirSync(srcDir).forEach(file => {
+  srcFile = srcDir + "/" + file;
+  dstFile = dstDir + "/" + file;
+  fs.createReadStream(srcFile).pipe(fs.createWriteStream(dstFile));
+  fs.chmodSync(srcFile , '755');
+});
```
Comment (rchougule) [inline scripts/copyhooks.js]: Refer the `path` core module here for path related operations...
forward slash might not work for windows etc.

`path.resolve` etc...

File: scripts/copyhooks.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+const fs = require('fs'); 
+
+const srcDir = './hooks/';
+const dstDir = './.git/hooks/'
+
+fs.readdirSync(srcDir).forEach(file => {
+  srcFile = srcDir + "/" + file;
+  dstFile = dstDir + "/" + file;
```
Comment (rchougule) [inline scripts/copyhooks.js]:
```suggestion
  const srcFile = srcDir + "/" + file;
  const dstFile = dstDir + "/" + file;
```

File: scripts/copyhooks.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+  fs.createReadStream(srcFile).pipe(fs.createWriteStream(dstFile));
+  fs.chmodSync(srcFile , '755');
```
Comment (rchougule) [inline scripts/copyhooks.js]:
```suggestion
  fs.chmodSync(dstFile , '755');
```

File: scripts/copyhooks.js (hunk 1 above)
Comment (neerajshukla1911) [reply scripts/copyhooks.js]: done

File: scripts/copyhooks.js [hunk 4]
```diff
@@ -0,0 +1,12 @@
+const fs = require('fs');
+const path = require('path');
+
+const srcDir = './hooks/';
+const dstDir = './.git/hooks/'
```
Comment (rchougule) [inline scripts/copyhooks.js:5]: Will these be environment agnostic?
should it be something like
```suggestion
const srcDir = path.resolve(__dirname, 'hooks');
const dstDir = path.resolve(__dirname, '.git', 'hooks');
```
please verify...

Comment (neerajshukla1911) [reply scripts/copyhooks.js:5]: yes, they are.

Comment (neerajshukla1911) [reply scripts/copyhooks.js:5]: Tested on windows, mac and linux.

Comment (rchougule) [reply scripts/copyhooks.js:5]: Okay. Can be resolved.

File: scripts/copyhooks.js (hunk 3 above)
Comment (neerajshukla1911) [reply scripts/copyhooks.js]: done

File: scripts/copyhooks.js (hunk 1 above)
Comment (rchougule) [reply scripts/copyhooks.js]: can be resolved

File: scripts/copyhooks.js (hunk 2 above)
Comment (rchougule) [reply scripts/copyhooks.js]: can be resolved

File: scripts/copyhooks.js (hunk 3 above)
Comment (rchougule) [reply scripts/copyhooks.js]: can be resolved

**Comment by rchougule** [comment]
Changes look good. Will approve post trying locally.

**Review by rchougule — APPROVED** [review]
LGTM

**Review by francisf — APPROVED** [review]

**Pull request merged**